tests/meta/demultiplex/test_validation.py

```diff
@@ -172,22 +173,29 @@ def test_is_flow_cell_ready_for_delivery_false(tmp_path: Path):
     assert result == False
 
 
-def test_validate_sample_sheet_exists_raises_error(tmp_path: Path):
-    # GIVEN a path with no sample sheet
-
+def test_validate_sample_sheet_exists_raises_error(bcl2fastq_flow_cell_dir: Path):
+    # GIVEN a flow cell without a sample sheet in housekeeper
+    flow_cell = FlowCellDirectoryData(flow_cell_path=bcl2fastq_flow_cell_dir)
+    flow_cell._sample_sheet_path_hk = None
     # WHEN validating the existence of the sample sheet
     # THEN it should raise a FlowCellError
     with pytest.raises(FlowCellError):
-        validate_sample_sheet_exists(tmp_path)
+        validate_sample_sheet_exists(flow_cell=flow_cell)
 
 
-def test_validate_sample_sheet_exists_no_error(tmp_path: Path):
+def test_validate_sample_sheet_exists_no_error(bcl2fastq_flow_cell_dir: Path):
```

Comment:
I think testing the correct functionality of the test should be just test_function, but feel free to disregard
```suggestion
def test_validate_sample_sheet_exists(bcl2fastq_flow_cell_dir: Path):
```